submissions/samuelJegede/database/transportation database.sql

```diff
@@ -0,0 +1,28 @@
+CREATE DATABASE transport;
+USE transport;
+CREATE TABLE passengers (
+	id INT NOT NULL AUTO_INCREMENT,
+    full_name VARCHAR(250) NOT NULL,
+    sex VARCHAR(6) NOT NULL,
+    age VARCHAR(3),
+    PRIMARY KEY (id)
+);
+CREATE TABLE trip_details (
+	id INT NOT NULL AUTO_INCREMENT,
+    passenger_id INT NOT NULL,
+    passenger_class ENUM('1', '2', '3') NOT NULL,
+    passenger_ticket_number VARCHAR(10) NOT NULL,
```

Comment:
Allow more characters. Dont manage the characters biko. You are dealing with data from a large database. So always allow for more characters.